### PR TITLE
Remove the K: Copy constraint, make it Clone instead

### PR DESCRIPTION
**Pull request by matthiasbeyer**
A possible solution for #9 

This is of course very hacky and everything, and possibly not the nicest solution. But as a proposal, it's an idea I guess.

### CODE REVIEW

File: src/panel.rs [hunk 1]
```diff
@@ -144,23 +144,23 @@ impl<K: Hash + Eq + Copy + Display + 'static> TabPanel<K> {
...
         self.bar.add_button_at(self.tx.clone(), id, pos);
         self
     }
 
     /// Remove a tab of the enclosed `TabView`.
     pub fn remove_tab(&mut self, id: K) -> Result<(), ()> {
-        self.bar.remove_button(id);
-        self.tabs.remove_tab(id)
+        self.bar.remove_button(&id);
+        self.tabs.remove_tab(&id)
```
Comment (jwuensche) [inline src/panel.rs:163]: I would say let's simply pass the reference to the function itself, this way we also behave similar as tab view already does.
So change the definition to something like this:
```rust
pub fn remove_tab(&mut self, id: &K) -> Result<(),()> {
```

**Comment by jwuensche** [comment]
I can also take care of the changes as they are mostly small adjustments, if you're okay with that and I'll add them to your branch.

**Comment by matthiasbeyer** [comment]
Fixed the merge conflict by rebasing and fixed your comment.

Examples work with `cargo check --examples --all` for me.

**Comment by jwuensche** [comment]
The branch looks good now, the tests run now without troubles.I'll merge them and we can release a new version today.
As for the hackiness of the solution, it's alright. We might want to restructure this code to reduce some of the clones, but we can do this later on.

Thanks so much for your work on this :smile:

**Comment by matthiasbeyer** [comment]
Awesome, thank you very much!

**Pull request merged**